src/components/StyleGuide/Forms.stories.mdx

```diff
@@ -6,7 +6,7 @@ import List from "../List/List";
 import { ListTypes } from "../List/ListTypes";
 import { getCategory } from "../../utils/componentCategories";
 
-<Meta title={getCategory("Forms")} />
+<Meta title={getCategory("Forms")} decorators={[withDesign]} />
```

Comment:
This is only needed if the figma file will be displayed in the canvas addon section, which in this case will not be displayed.